scripts/globals/summon.lua

```diff
@@ -1,241 +1,187 @@
     if (shadowbehav < 5 and shadowbehav ~= MOBPARAM_IGNORE_SHADOWS) then --remove 'shadowbehav' shadows.
-        targShadows = target:getMod(dsp.mod.UTSUSEMI);
-        shadowType = dsp.mod.UTSUSEMI;
+        targShadows = target:getMod(dsp.mod.UTSUSEMI)
+        shadowType = dsp.mod.UTSUSEMI
         if (targShadows==0) then --try blink, as utsusemi always overwrites blink this is okay
-            targShadows = target:getMod(dsp.mod.BLINK);
-            shadowType = dsp.mod.BLINK;
+            targShadows = target:getMod(dsp.mod.BLINK)
+            shadowType = dsp.mod.BLINK
         end
 
         if (targShadows>0) then
         -- Blink has a VERY high chance of blocking tp moves, so im assuming its 100% because its easier!
             if (targShadows >= shadowbehav) then --no damage, just suck the shadows
-                skill:setMsg(dsp.msg.basic.SHADOW_ABSORB);
-                target:setMod(shadowType,(targShadows-shadowbehav));
+                skill:setMsg(dsp.msg.basic.SHADOW_ABSORB)
+                target:setMod(shadowType,(targShadows-shadowbehav))
                 if (shadowType == dsp.mod.UTSUSEMI) then --update icon
-                    effect = target:getStatusEffect(dsp.effect.COPY_IMAGE);
+                    effect = target:getStatusEffect(dsp.effect.COPY_IMAGE)
                     if (effect ~= nil) then
                         if ((targShadows-shadowbehav) == 0) then
-                            target:delStatusEffect(dsp.effect.COPY_IMAGE);
-                            target:delStatusEffect(dsp.effect.BLINK);
+                            target:delStatusEffect(dsp.effect.COPY_IMAGE)
+                            target:delStatusEffect(dsp.effect.BLINK)
                         elseif ((targShadows-shadowbehav) == 1) then
-                            effect:setIcon(dsp.effect.COPY_IMAGE);
+                            effect:setIcon(dsp.effect.COPY_IMAGE)
                         elseif ((targShadows-shadowbehav) == 2) then
-                            effect:setIcon(dsp.effect.COPY_IMAGE_2);
+                            effect:setIcon(dsp.effect.COPY_IMAGE_2)
                         elseif ((targShadows-shadowbehav) == 3) then
-                            effect:setIcon(dsp.effect.COPY_IMAGE_3);
+                            effect:setIcon(dsp.effect.COPY_IMAGE_3)
                         end
                     end
                 end
-                return shadowbehav;
+                return shadowbehav
             else -- less shadows than this move will take, remove all and factor damage down
-                dmg = dmg * ((shadowbehav-targShadows)/shadowbehav);
-                target:setMod(dsp.mod.UTSUSEMI,0);
-                target:setMod(dsp.mod.BLINK,0);
-                target:delStatusEffect(dsp.effect.COPY_IMAGE);
-                target:delStatusEffect(dsp.effect.BLINK);
+                dmg = dmg * ((shadowbehav-targShadows)/shadowbehav)
```

Comment:
Space out the operators and the outside set of parentheses are unneeded. (Keep the ones around the subtraction)